recipes/dpp/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get, copy
```

Comment:
```suggestion
from conan.tools.files import get, copy, rmdir
```

Reply:
thanks, i'll merge these, first i want to see how far the conan 2 build gets